libevm/VMCalls.cpp

```diff
@@ -120,14 +120,21 @@ void VM::caseCreate()
 	uint64_t initOff = (uint64_t)m_SP[1];
 	uint64_t initSize = (uint64_t)m_SP[2];
 
+	// Clear the return data buffer. This will not free the memory.
+	m_returnData.clear();
+
 	if (m_ext->balance(m_ext->myAddress) >= endowment && m_ext->depth < 1024)
 	{
 		*m_io_gas_p = m_io_gas;
 		u256 createGas = *m_io_gas_p;
 		if (!m_schedule->staticCallDepthLimit())
 			createGas -= createGas / 64;
 		u256 gas = createGas;
-		m_SPP[0] = (u160)m_ext->create(endowment, gas, bytesConstRef(m_mem.data() + initOff, initSize), m_onOp);
+		h160 addr;
+		owning_bytes_ref revertOutput;
```

Comment:
Maybe name it just `output`? Because `revertOutput` is kind of assuming how `create` works internally and that only `REVERT` can return data. Which is the case now, but could change in the furure